plugins/install-helm.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: install-helm
+spec:
+  platforms:
+  - head: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    sha256: c548f94d8efd1fb8aa35a54b7e5cdb64ae012447122d9704f3cee4163bbbe9c3
+    uri: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    files:
+    - from: "/*/install-helm/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.1"
+  shortDescription: Install helm package manager
```

Comment:
Similarly, plugins ideally should not be one-off use like installing Helm.

If there's something like `curl https://helm.sh/install.sh | bash`, users should use that.

Reply:
This one is really useful for me because it bundles all of the commands for installing helm with RBAC.

Reply:
Is there a rubric or documentation with guidelines on the usefulness of a plugin so that it can be admitted to the Krew-index?

Reply:
Is the RBAC problem because GKE does not automatically create `cluster-admin` binding to your account (I believe this will be fixed soon) OR is it because helm installation is not creating the `tiller` service account for you?

Reply:
it's both the cluster-admin binding and the tiller service-account and rbac role.